build/ci.sh

```diff
@@ -1,3 +1,9 @@
 set -e  # Exit immediately if a command exits with a non-zero status.
 
 npm run lint
+npm run test:cover
```

Comment:
Should we have npm run test:unit as well? Or just npm run test for running all the unit and coverage tests?